scaling_performance/optimizing_ephemeral_storage.adoc

```diff
@@ -0,0 +1,75 @@
+[[scaling-performance-optimizing-ephemeral-storage]]
+= Optimizing Ephemeral Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+*This page only applies if the Ephemeral Storage tech preview in
+{product-title} 3.10 has been enabled.  This is disabled by default;
+administrators may enable it via the instructions in
+xref:../install_config/configuring_ephemeral.adoc[Configuring for
+Ephemeral Storage].*
+
+Ephemeral storage is used by pods for their internal operation, such
+as for temporary files.  The lifetime of this ephemeral storage does
+not extend beyond the life of the individual pod, and this ephemeral
+storage cannot be shared across pods.
+
+Prior to {product-title} 3.10, ephemeral local storage was already
+exposed to pods via the container’s writable layer, logs directory,
+and EmptyDir volumes.  There are many issues related to the lack of
+local storage accounting and isolation, including:
+
+- Pods do not know how much local storage is available to them.
+
+- Pods cannot request "guaranteed" local storage.
+
+- Local storage is a "best effort" resource.
+
+- Pods can get evicted due to other pods filling up the local storage,
+after which no new pods will be admitted until sufficient storage has
+been reclaimed.
+
+Ephemeral storage is still exposed to pods in the same way, but there
+are new methods for implementing requests and limits on pods'
+consumption of ephemeral storage.
+
+Note that management of container logs apply only if you are using
+CRI-O as the container runtime and use file-based logging.
+
+It is important to understand that ephemeral storage is shared among
+all pods in the system, and that {product-title} does not provide any
+mechanism for guaranteeing any level of service beyond the requests
+and limits established by the administrator and users.  For example,
+ephemeral storage does not provide any guarantees of throughput, I/O
+operations/second, or any other measure of storage performance.
+
+== General Storage Guidelines
```

Comment:
Not really "General storage guidelines". This is more about the location of a node's local storage. The only thing relevant to ephemeral storage here is that only primary partitions can be used for ephemeral local storage -- can the user do anything to actually control/manage this, or is this simply a design/conceptual detail that may not even be useful to the user?

Reply:
The heading should be "General ephemeral storage guidelines" or the like.

The user can't control this, but it's relevant to the administrator in terms of where to put the primary partition.  By default it's on the root filesystem, but the administrator can choose to put it elsewhere.

Reply:
I think "General ephemeral storage guidelines" is a better heading.

Reply:
OK

Reply:
All pushed and ready for re-review.